pkg/querier/distributor_queryable.go

```diff
@@ -54,6 +70,45 @@ func (q *distributorQuerier) Select(sp *storage.SelectParams, matchers ...*label
 	return matrixToSeriesSet(matrix), nil, nil
 }
 
+func (q *distributorQuerier) streamingSelect(sp storage.SelectParams, matchers []*labels.Matcher) (storage.SeriesSet, storage.Warnings, error) {
```

Comment:
Note to myself: except for the comment left (about labels passed to `chunkcompact.FromChunks`), this is the exact copy of `ingesterStreamingQuerier.Select()`.